### PR TITLE
UAT script initial functionality

### PR DESCRIPTION
**Pull request by Thomas-S-Allen**

### CODE REVIEW

File: quick_classifier.py [hunk 1]
```diff
@@ -69,7 +97,7 @@ def bottom_k_scores(scores, k):
 
     if args.records:
         records_path = args.records
-        out_path = records_path.replace('.csv', 'uat_keywords.tsv')
+        out_path = records_path.replace('.csv', '_uat_keywords.tsv')
```
Comment (tjacovich) [inline quick_classifier.py:100]: Make this file name configurable.

File: quick_classifier.py [hunk 2]
```diff
@@ -136,11 +164,14 @@ def bottom_k_scores(scores, k):
                 keyword_list = [kw['category'] for kw in keywords]
                 score_list = [kw['score'] for kw in keywords]
 
-
-            # header = 'bibcode,title,abstract,keywords,scores,threshold'
-            record_output = [record['bibcode'],record['title'],record['abstract'],', '.join(map(str, keyword_list)),', '.join(map(str, score_list)),str(threshold)]
-
-            output_list.append(record_output)
+                for keyword in keywords:
+                    print()
```
Comment (tjacovich) [inline quick_classifier.py:168]: Remove prints from final commit

File: quick_classifier.py [hunk 3]
```diff
@@ -79,7 +107,7 @@ def bottom_k_scores(scores, k):
     output_idx = 0
     output_list = []
     output_batch = 500
-    header = 'bibcode,title,abstract,keywords,scores,threshold'
+    header = 'bibcode,uat_branch'
```
Comment (tjacovich) [inline quick_classifier.py:110]: Add UAT number

File: quick_classifier.py [hunk 4]
```diff
@@ -50,6 +50,34 @@ def top_k_scores(scores, k):
         
 def bottom_k_scores(scores, k):
     return(heapq.nsmallest(k, scores, key=lambda x: x['score']) )
+
+def get_keyword_hierarchy(keyword):
+    hierarchy_paths = []
+
+    # import pdb;pdb.set_trace()
```
Comment (tjacovich) [inline quick_classifier.py:57]: Drop pdb lines from final commit

**Review by tjacovich — CHANGES_REQUESTED** [review]
Update help comment for input file and then the other little clean-up comments and this should be good to go!

File: quick_classifier.py (hunk 1 above)
Comment (Thomas-S-Allen) [reply quick_classifier.py]: Made output filename configurable in config.py.  Also, added command line arguments, `-p` or `--preserve_filename`, to preserve naming based on input filename.

File: quick_classifier.py (hunk 3 above)
Comment (Thomas-S-Allen) [reply quick_classifier.py]: Header for output filename changed to 'bibcode,uat_branch,uat_numner'.

File: quick_classifier.py (hunk 2 above)
Comment (Thomas-S-Allen) [reply quick_classifier.py]: Print lines cleaned up.  Command line option (`-v` or `--verbose`) added to allow more detailed, record level information to be printed to screen.

**Pull request merged**